doc/reference/devices_nic.md

```diff
@@ -1,410 +1,454 @@
 (devices-nic)=
 # Type: `nic`
 
-LXD supports several different kinds of network devices (referred to as Network Interface Controller or NIC).
+```{note}
+The `nic` device type is supported for both containers and VMs.
 
-When adding a network device to an instance, there are two ways to specify the type of device you want to add;
-either by specifying the `nictype` property or using the `network` property.
-
-## Specifying a NIC using the `network` property
-
-When specifying the `network` property, the NIC is linked to an existing managed network and the `nictype` is
-automatically detected based on the network's type.
-
-Some of the NICs properties are inherited from the network rather than being customizable for each NIC.
-
-These are detailed in the "Managed" column in the NIC specific sections below.
+Most NICs support hotplugging for both containers and VMs.
+See the description for each NIC type for exceptions.
+```
 
-## NICs Available
+Network devices, also referred to as *Network Interface Controllers* or *NICs*, supply a connection to a network.
+LXD supports several different types of network devices (*NIC types*).
 
-See the settings for the NIC below for details about which properties are available.
+## `nictype` vs. `network`
 
-The following NICs can be specified using the `nictype` or `network` properties:
+When adding a network device to an instance, there are two methods to specify the type of device that you want to add: through the `nictype` device option or the `network` device option.
 
-- [`bridged`](#nic-bridged): Uses an existing bridge on the host and creates a virtual device pair to connect the host bridge to the instance.
-- [`macvlan`](#nic-macvlan): Sets up a new network device based on an existing one but using a different MAC address.
-- [`sriov`](#nic-sriov): Passes a virtual function of an SR-IOV enabled physical network device into the instance.
+These two device options are mutually exclusive, and you can specify only one of them when you create a device.
+However, note that when you specify the `network` option, the `nictype` option is set automatically.
 
-The following NICs can be specified using only the `network` property:
+`nictype`
+: When using the `nictype` device option, you specify a network interface that is not controlled by LXD.
```

Comment:
This isn't strictly true. You can use `nictype` and `parent` and reference a LXD managed bridge, for instance.

Perhaps add a "can" to that sentence instead:

> When using the `nictype` device option, you can specify a network interface that is not controlled by LXD.